README.md

```diff
@@ -220,6 +220,13 @@ configuration files, passthru, support for searching compressed files,
 multiline search and opt-in fancy regex support via PCRE2.
 
 
+### Playground
+
+If you'd like to try ripgrep before installing, there's a [playground](https://codapi.org/ripgrep/) and an [interactive tutorial](https://codapi.org/try/ripgrep/).
+
+If you have any questions about these, please open an issue in the [tutorial repo](https://github.com/nalgeon/tryxinyminutes).
```

Comment:
Could you add the word "unofficial" somewhere in this text. Maybe, "there's an unofficial playground." Just so folks are clear that it isn't maintained as part of the ripgrep project.

Also, could you follow the stylistic convention of the surrounding text and wrap lines to 79 columns (inclusive)?

Reply:
Sure!